src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
```

Comment:
change the type from below this line to above this line

Reply:
Someone should look into the examples as it seems every project has "below" where it should be "above"